Backend/core/comms.py

```diff
@@ -217,6 +218,24 @@ def parse_packets(self, new_data: bytes, tmp_source: str):
 
         return packets
 
+    def fs_down_callback(self, data):
+        # copied from listen_upd()
+        if not data:
+            # No data received, we're done here
+            return
+
+        packets = self.parse_packets(data, 'file_sync')
+        for packet in packets:
+            if len(packet) == byte_length:
+                d = unpack_data(packet)
+                frontend_data = d.copy()
```

Comment:
Let's not set the frontend_data for file sync since it's not live data

Reply:
Good call

Reply:
Especially good call because I actually didn't include the `global` line like in the UDP method, so it was just creating a pointless local variable